### PR TITLE
Upgrade to n-express + n-logger

### PR DESCRIPTION
**Pull request by matthew-andrews**
@andygnewman

### CODE REVIEW

**Comment by matthew-andrews** [comment]
This brings all our prod NPM dependencies up to date except for `ft-next-article-primary-tag`.

File: package.json [hunk 1]
```diff
@@ -7,19 +7,17 @@
...
-    "cheerio": "~0.19.0",
-    "eslint": "^1.10.3",
-    "fetchres": "~1.4.0",
-    "forever": "~0.15.1",
+    "cheerio": "^0.20.0",
+    "fetchres": "^1.4.0",
+    "forever": "^0.15.1",
     "ft-n-article-branding": "^3.0.0",
     "ft-next-article-genre": "^1.1.0",
     "ft-next-article-primary-tag": "^1.0.0",
```
Comment (andygnewman) [inline package.json]: ft-next-article-primary-tag is also out of date (v2 available) however is unused in the code base, so can just be deleted.

**Comment by andygnewman** [comment]
👍 , but can you leave merging this until after the dev huddle on Friday, so I can demo Whitesource using article as an example - once you do this, it will clear all the alerts.

**Comment by matthew-andrews** [comment]
yup that's fine — you can use this pr to demo clearing the alerts, if you like 😄

**Pull request merged**